page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/DefaultPage.java

```diff
@@ -0,0 +1,168 @@
+     * @throws ru.sbtqa.tag.pagefactory.exceptions.PageException if required
+     * element couldn't be found, or current page isn't initialized
+     */
+    @ActionTitle("ru.sbtqa.tag.pagefactory.select")
+    public void select(String elementTitle, String option) throws PageException {
+        Object element = ReflectionUtils.getElementByTitle(PageContext.getCurrentPage(), elementTitle);
+        pageActions.select(element, option);
+    }
+
+    /**
+     * Find web element with corresponding title, if it is a check box, select
+     * it If it's a WebElement instance, check whether it is already selected,
+     * and click if it's not Add corresponding parameter to allure report
+     *
+     * @param elementTitle WebElement that is supposed to represent checkbox
+     * @throws ru.sbtqa.tag.pagefactory.exceptions.PageException if page was not
+     * initialized, or required element couldn't be found
+     */
+    @ActionTitle("ru.sbtqa.tag.pagefactory.select.checkbox")
+    public void setCheckBox(String elementTitle) throws PageException {
+        Object element = ReflectionUtils.getElementByTitle(PageContext.getCurrentPage(), elementTitle);
+        pageActions.setCheckbox(element, true);
+    }
+
+
+    /**
+     * TODO
+     *
+     * @param text string value that will be searched inside of the element
+     * @param elementTitle title of the element to search
+     * @throws ru.sbtqa.tag.pagefactory.exceptions.ElementNotFoundException if
+     * couldn't find element by given title, or current page isn't initialized
+     */
+    @ActionTitle("ru.sbtqa.tag.pagefactory.check.value")
+    public void checkValueIsEqual(String elementTitle, String text) throws PageException {
+        Object element = ReflectionUtils.getElementByTitle(PageContext.getCurrentPage(), elementTitle);
+        if (!pageChecks.checkEquality(element, text)) {
+            throw new AutotestError("'" + elementTitle + "' value is not equal with '" + text + "'");
+        }
+    }
+
+    /**
+     * Find element with corresponding title, and make sure that its value is
+     * not equal to given text Text, as well as element value are being
+     * space-trimmed before comparison, so only non-space characters matter
+     *
+     * @param text element value for comparison
+     * @param elementTitle title of the element to search
+     * @throws ru.sbtqa.tag.pagefactory.exceptions.PageException if current page
+     * wasn't initialized, or element with required title was not found
+     */
+    @ActionTitle("ru.sbtqa.tag.pagefactory.check.values.not.equal")
+    public void checkValueIsNotEqual(String text, String elementTitle) throws PageException {
+        Object element = ReflectionUtils.getElementByTitle(PageContext.getCurrentPage(), elementTitle);
+        if (pageChecks.checkEquality(element, text)) {
+            throw new AutotestError("'" + elementTitle + "' value is equal with '" + text + "'");
+        }
+    }
+
+    /**
+     * TODO
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)